do.sh

```diff
@@ -207,6 +231,7 @@ ovn_fmn_repo="${OVN_FAKE_MULTINODE_REPO:-https://github.com/ovn-org/ovn-fake-mul
 ovn_fmn_branch="${OVN_FAKE_MULTINODE_BRANCH:-main}"
 
 OS_IMAGE_OVERRIDE="${OS_IMAGE_OVERRIDE}"
+OS_IMAGE_DEFAULT="registry.fedoraproject.org/fedora:38"
```

Comment:
Before it was "whatever ovn-fake-multinode builds by default", which is `fedora:latest`.  Should we use `fedora:latest` here?

Reply:
It makes sense.  I changed it back to `fedora:latest`.